ee/agent/flags/keys/keys.go

```diff
@@ -57,6 +57,7 @@ const (
 	LocalDevelopmentPath            FlagKey = "localdev_path"
 	LauncherWatchdogEnabled         FlagKey = "launcher_watchdog_enabled" // note that this will only impact windows deployments for now
 	SystrayRestartEnabled           FlagKey = "systray_restart_enabled"
+	OsqueryVersion                  FlagKey = "osquery_version"
```

Comment:
I think I would make this `running_osquery_version` or `current_osquery_version` a) for consistency with your `SetCurrrentRunningOsqueryVersion` method naming, and b) to indicate that this flag reflects but does not control the osquery version. (It's fine to leave the log field as `osquery_version` in logshipper.go -- I think it only matters here, where it's a control server flag.)

What do other folks think?

Reply:
Renamed to "running_osquery_version"